metricbeat/module/mysql/query/query.go

```diff
@@ -15,12 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
-/*
-Package status fetches MySQL server status metrics.
+//go:build !requirefips
 
-For more information on the query it uses, see:
-http://dev.mysql.com/doc/refman/5.7/en/show-status.html
-*/
```

Comment:
This is copy-pasta mistake in earlier commits. Removed.

Reply:
in a different PR? can you link it?